universal-login-ops/src/dev/startRelayer.ts

```diff
@@ -0,0 +1,17 @@
+import Relayer, {DevelopmentRelayer} from 'universal-login-relayer';
+import {providers} from 'ethers';
+
+export type RelayerConstructor = {
+  new (config: any, provider: providers.Provider): DevelopmentRelayer | Relayer;
```

Comment:
Should be enough to do simpler type:
`new (config: any, provider: providers.Provider): Relayer;`, as
`DevelopmentRelayer` inherits from `Relayer`